docs/source/examples/Layout and Styling.ipynb

```diff
@@ -0,0 +1,1892 @@
+    "\n",
+    "   This defines the default behaviour for how flex items are laid out along the cross axis on the current line. Think of it as the justify-content version for the cross-axis (perpendicular to the main-axis).\n",
+    "   ![Items](./images/align-items.svg)\n",
+    "   \n",
+    "- `align-content` (flex-start | flex-end | center | baseline | stretch)\n",
+    "\n",
+    "   This aligns a flex container's lines within when there is extra space in the cross-axis, similar to how justify-content aligns individual items within the main-axis.\n",
+    "   ![Items](./images/align-content.svg)\n",
+    "\n",
+    "  **Note**: this property has no effect when there is only one line of flex items.\n",
+    "\n",
+    "### Properties of the items\n",
+    "\n",
+    "![Item](./images/flex-items.svg)\n",
+    "\n",
+    "The flexbox-related css properties of the items have no impact if the parent element is not a flexbox container (i.e. has a `display` attribute equal to `flex` or `inline-flex`).\n",
+    "\n",
+    "\n",
+    "- `order`\n",
+    "\n",
+    "   By default, flex items are laid out in the source order. However, the order property controls the order in which they appear in the flex container.\n",
+    "   <img src=\"./images/order-2.svg\" alt=\"Order\" style=\"width: 500px;\"/>\n",
+    "   \n",
+    "- `flex` **(shorthand for three properties)**\n",
+    "   This is the shorthand for flex-grow, flex-shrink and flex-basis combined. The second and third parameters (flex-shrink and flex-basis) are optional. Default is `0 1 auto`.\n",
+    "   \n",
+    "    - `flex-grow`\n",
+    "    \n",
+    "      This defines the ability for a flex item to grow if necessary. It accepts a unitless value that serves as a proportion. It dictates what amount of the available space inside the flex container the item should take up.\n",
+    "\n",
+    "      If all items have flex-grow set to 1, the remaining space in the container will be distributed equally to all children. If one of the children a value of 2, the remaining space would take up twice as much space as the others (or it will try to, at least).\n",
+    "        ![Grow](./images/flex-grow.svg)\n",
+    "      \n",
+    "    - `flex-shrink`\n",
+    "    \n",
+    "      This defines the ability for a flex item to shrink if necessary.\n",
+    "      \n",
+    "    - `flex-basis`\n",
+    "    \n",
+    "      This defines the default size of an element before the remaining space is distributed. It can be a length (e.g. `20%`, `5rem`, etc.) or a keyword. The `auto` keyword means *\"look at my width or height property\"*.\n",
+    "      \n",
+    "- `align-self`\n",
+    "\n",
+    "  This allows the default alignment (or the one specified by align-items) to be overridden for individual flex items.\n",
+    "\n",
+    "  ![Align](./images/align-self.svg)\n",
+    "\n",
+    "### The VBox and HBox helpers\n",
```

Comment:
The images in the two sections above this, `Properties of the parent` and `Properties of the items` do not render for me on GitHub... Maybe this is just a GH problem?